platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/system/SystemExitCode.java

```diff
@@ -18,6 +18,10 @@
 
 public enum SystemExitCode {
     NO_ERROR(0),
+    /**
+     * Configuration error.
+     */
+    CONFIGURATION_ERROR(1),
```

Comment:
mostly a nit, but exit code 1 may be interpreted specifically.
Generally, application-specific errors should avoid "common" exit codes, which include
1, 2, 126, 127, 128, 255, and 64-78.
Note that 129-254 are interpreted as "fatal error number (code - 128)" and can be caused by signals and other O/S actions that end the process (e.g. `kill {process}` would produce exit code 143), which can lead to confusion with application codes.
The ideal range for application codes is 33-63, which are the least likely to be used by the O/S or shell.
`33` might be good for configuration error.

Reply:
changing to 33.